P5/Source/Specs/facsimile.xml

```diff
@@ -28,13 +28,35 @@ ou encodé.</desc>
     <sequence>
       <elementRef key="front" minOccurs="0"/>
       <alternate minOccurs="1" maxOccurs="unbounded">
-	<classRef key="model.graphicLike"/>
-	<elementRef key="surface"/>
-	<elementRef key="surfaceGrp"/>
+        <classRef key="model.graphicLike"/>
+        <elementRef key="surface"/>
+        <elementRef key="surfaceGrp"/>
       </alternate>
       <elementRef key="back" minOccurs="0"/>
     </sequence>
   </content>
+  <constraintSpec scheme="schematron" ident="no_facsimile_text_nodes">
+    <constraint>
+      <sch:rule context="tei:facsimile//tei:line | tei:facsimile//tei:zone">
+        <sch:report test="child::text()[ normalize-space(.) ne '']">
+  	  A facsimile element represents a text with images, thus
+	  transcribed text should not be present within it.
+	</sch:report>
+        <!-- 
```

Comment:
I suggest to remove the comment. an issue has already been opened.

Reply:
@martinascholger It's issue #2363, I believe? (for the record)

My 2 cents here are that the comment doesn't hurt, yet I think it's helpful and can be removed when issue #2363 is done